common/changes/@itwin/appui-layout-react/as-eslint-upgrade_2023-07-11-20-20.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/appui-layout-react",
+      "comment": "Upgrade @itwin/eslint-plugin to 4.0.0-dev.43 and eslint to 8.44.0",
```

Comment:
for all changelogs
these are what the users see, they shouldnt have to know which exact version or see mention of a pre-release 

```suggestion
      "comment": "Upgrade to eslint@8 and @itwin/eslint-plugin accordingly",
```

Reply:
@aruniverse I fell there should be no comment at all, this is internal dev changes only, not for consumers, any reason to add a changelog comment at all for this ?

Reply:
good point, i'm ok not having a comment at all for the changelogs

Reply:
updated